src/context/directory/handlers/branding.ts

```diff
@@ -10,17 +10,24 @@ type ParsedBranding = ParsedAsset<'branding', Asset>;
 
 function parse(context: DirectoryContext): ParsedBranding {
   const brandingDirectory = path.join(context.filePath, constants.BRANDING_DIRECTORY);
+  const brandingFile = path.join(brandingDirectory, 'branding.json');
 
   if (!existsMustBeDir(brandingDirectory)) return { branding: null };
 
-  const branding = loadJSON(path.join(brandingDirectory, 'branding.json'), context.mappings);
+  const branding = (() => {
+    if (isFile(brandingFile)) {
+      return loadJSON(brandingFile, context.mappings);
+    }
+
+    return null;
+  })();
 
   const brandingTemplatesFolder = path.join(
     brandingDirectory,
     constants.BRANDING_TEMPLATES_DIRECTORY
   );
 
-  if (!existsMustBeDir(brandingTemplatesFolder)) return { branding: null };
+  if (!existsMustBeDir(brandingTemplatesFolder)) return { branding: branding };
```

Comment:
**Nitpick:**
```suggestion
  if (!existsMustBeDir(brandingTemplatesFolder)) return { branding };
```

Reply:
by modifying the variable above to `brandingSettings` this has to be `branding: brandingSettings`.